insights/specs/datasources/get_semid.py

```diff
@@ -0,0 +1,26 @@
+from insights.core.context import HostContext
+from insights.core.plugins import datasource
+from insights.parsers.ipcs import IpcsS
```

Comment:
If there is possible to get the expected list from an existing spec, do not use the Parser which is heavier.

The existing spec `ipcs_s` is quite simple, and can be used directly for this datasource.